src/db/h2/lrsql/h2/sql/query.sql

```diff
@@ -268,6 +275,14 @@ SELECT 1 FROM admin_account
 --~ (when (:username params)   "WHERE username = :username")
 --~ (when (:account-id params) "WHERE id = :account-id")
 
+-- :name query-account-count-local
+-- :command :query
+-- :result :one
+-- :doc Count the local admin accounts present.
```

Comment:
Maybe make this more clear that it does _not_ count OIDC accounts ("local accounts" does indicate that but it doesn't hurt to be more explicit).

Reply:
https://github.com/yetanalytics/lrsql/pull/294/commits/99e44ad5cebde584458be4d158c11bdda9a97f1f